localenv/cloud-nine-wallet/docker-compose.yml

```diff
@@ -74,6 +74,7 @@ services:
       WALLET_ADDRESS_URL: ${CLOUD_NINE_WALLET_ADDRESS_URL:-https://cloud-nine-wallet-backend/.well-known/pay}
       ILP_CONNECTOR_URL: ${CLOUD_NINE_CONNECTOR_URL:-http://cloud-nine-wallet-backend:3002}
       ENABLE_TELEMETRY: true
+      ENABLE_TELEMETRY_TRACES: true
```

Comment:
I think since we have this in the telemetry docker compose we can keep it turned off here IMO